### PR TITLE
hero_tywin.json

### PR DESCRIPTION
**Pull request by Zordas**
Tywin Hero Summary

### CODE REVIEW

File: hero/Tywin.json [hunk 1]
```diff
@@ -0,0 +1,199 @@
...
+        "Captain of the Royal Guard of Lefundos, situated in Northern Eureka, Tywin holds the favor of his king Schniel. A hard worker, he values rule and order over all else, and gives off the impression that he will not listen to what you have to say."
+    ],
+    "relations": [
+        { "hero": "romann", "relationType": "trust" },
+        { "hero": "cecilia", "relationType": "trust" },
+        { "hero": "otillie", "relationType": "trust" }
+    ],
+    "stats": {
+        "base": {
+            "cp": 2520,
```
Comment (kmalone86) [inline hero/Tywin.json]:
```suggestion
            "cp": 2502,
```

File: hero/Tywin.json [hunk 2]
```diff
@@ -0,0 +1,199 @@
...
+        {
+            "isPassive": false,
+            "soulBurn": 0,
+            "selfSkillBarValue": 0,
+            "soulBurnEffect": "",
+            "awakenUpgrade": false,
+            "cooldown": 0,
+            "name": "Sword Storm",
+            "soulAcquire": 1,
+            "description": "Attacks with a sword storm, with a 50% chance to dispell one buff. Damage dealt increases proportional to the caster's max Health.",
```
Comment (kmalone86) [inline hero/Tywin.json]:
```suggestion
            "description": "Attacks with a swordstorm, with a 50% chance to dispel one buff. Damage dealt increases proportional to the caster's max Health.",
```

File: hero/Tywin.json [hunk 3]
```diff
@@ -0,0 +1,199 @@
...
+            "enhancement": [
+                "+5% damage dealt",
+                "+5% damage dealt",
+                "+10% effect chance",
+                "+10% damage dealt",
+                "+15% effect chance",
+				"+15% damage dealt"
```
Comment (kmalone86) [inline hero/Tywin.json]:
```suggestion
                "+15% damage dealt"
```

File: hero/Tywin.json [hunk 4]
```diff
@@ -0,0 +1,199 @@
...
+        },
+        {
+            "isPassive": false,
+            "soulBurn": 0,
+            "soulBurnEffect": "",
+            "awakenUpgrade": true,
+            "cooldown": 4,
+            "name": "Commanding Shout",
+            "soulAcquire": 1,
+            "description": "Grants increased attack and Critical Hit Chance to the caster and an ally for 2 turns, and increases combat readiness by 30%.",
```
Comment (kmalone86) [inline hero/Tywin.json]:
```suggestion
            "description": "Grants increased Attack and Critical Hit Chance to the caster and an ally for 2 turns, and increases Combat Readiness by 30%.",
```

File: hero/Tywin.json [hunk 5]
```diff
@@ -0,0 +1,199 @@
...
+            "enhancement": [
+                "Acquire +1 soul",
+                "-1 turn cooldown"
+            ],
+            "buffs": [{ "stic_atk_up": "" }, { "stic_chc_up": "" }],
```
Comment (kmalone86) [inline hero/Tywin.json]:
```suggestion
            "buffs": ["stic_atk_up", "stic_chc_up"],
```

File: hero/Tywin.json [hunk 6]
```diff
@@ -0,0 +1,199 @@
...
+            "debuffs": []
+        },
+        {
+            "isPassive": false,
+			"soulBurn": 20,
```
Comment (kmalone86) [inline hero/Tywin.json]:
```suggestion
            "soulBurn": 20,
```

File: hero/Tywin.json [hunk 7]
```diff
@@ -0,0 +1,199 @@
...
+            "soulBurnEffect": "Grants an extra turn.",
+            "awakenUpgrade": false,
+            "cooldown": 5,
+            "name": "All-Out Attack",
+            "soulAcquire": 2,
+            "description": "Attacks all enemies again after a swordstorm, with a 50% chance to decrease defense for 2 turns and increase defense of the caster for 3 turns. Damage dealt increases proportional to the caster's max Health. ",
```
Comment (kmalone86) [inline hero/Tywin.json]:
```suggestion
            "description": "Attacks all enemies again after a swordstorm, with a 50% chance to decrease Defense for 2 turns and increase Defense of the caster for 3 turns. Damage dealt increases proportional to the caster's max Health. ",
```

File: hero/Tywin.json [hunk 8]
```diff
@@ -0,0 +1,199 @@
...
+            "enhancement": [
+                "+5% damage dealt",
+                "+5% damage dealt",
+                "+10% effect chance",
+                "-1 turn cooldown",
+                "+15% effect chance",
+				"+10% damage dealt",
```
Comment (kmalone86) [inline hero/Tywin.json]:
```suggestion
                "+10% damage dealt",
```

File: hero/Tywin.json [hunk 9]
```diff
@@ -0,0 +1,199 @@
...
+				"+15% damage dealt"
```
Comment (kmalone86) [inline hero/Tywin.json]:
```suggestion
                "+15% damage dealt"
```

**Comment by Triharder-E7** [comment]
The file needs to be lowercase as well, but I don't think that's something I can change with a commit.

File: hero/Tywin.json (hunk 2 above)
Comment (kmalone86) [reply hero/Tywin.json]: New version still has "dispell" instead of "dispel".

**Pull request merged**